### DISCUSSION TITLE
[Desktop] How to draw Control using Angle?

### DISCUSSION THREAD

**Discussion by kucint** [unoplatform/uno#16909] [Q&A] [answered]
I am reconsidering to create couple of controls that perform 3D graphics.
Could you please provide simple code snippet that creates a (user?) control that performs direct drawing using [Angle](https://github.com/google/angle)?

Shall work on Uno Desktop apps (win/linux/macos).

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the question. This is currently work in progress: https://github.com/unoplatform/uno/pull/16621
